packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -241,13 +241,13 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
-  if (input?.price || input.metadata?.apps?.stripe?.price) {
-    data.price = input.price || input.metadata?.apps?.stripe?.price;
+  if (input.metadata?.apps?.stripe?.price) {
+    data.price = input.metadata?.apps?.stripe?.price;
     const paymentCredential = await ctx.prisma.credential.findFirst({
       where: {
         userId: ctx.user.id,
         type: {
-          contains: "_payment",
+          equals: "stripe_payment",
```

Comment:
Will this trigger for PayPal payments as well?

Reply:
See below, I didn't like this duplicated code but should work for a starter.